### ISSUE TITLE
Future TODO on the PEP 7-compliant .clang-format

### ISSUE THREAD

**Issue by hajimes** [hajimes/mmh3#61]
Not sure why, but the clang-format lint is failing on super-linter 7.0.0 (which uses clang-format 17.0.6) with the following format file: https://github.com/hajimes/mmh3/blob/e521390ac1c1b9cafa918ba91559497314d4b901/.clang-format.

This file is meant for PEP 7-compliant code and was originally written by Paul Ganssle. It's also used by Numpy with a few modifications.

This shouldn't be an issue with clang-format 18.1.8. Since strict PEP 7 compliance isn't required for our final production, I'm removing the `.clang-format` file for now. I'll check back and see if it works again when a future version of super-linter, which supports clang-format 18.*, is released.

**Comment by hajimes** [comment]
This failure occurred because, in super-linter 7.0.0, the default directory for linter configuration rules is `.github/linters`.  Setting the environment variable `LINTER_RULES_PATH` as `/` resolved the problem.

```yaml
      # Runs the Super-Linter action
      - name: Run Super-Linter
        uses: super-linter/super-linter@v7.0.0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          LINTER_RULES_PATH: /
```

**Comment by hajimes** [comment]
Fixed: c9dd7cdfcf859ce8e311d1487835c17ea5b6b53b.
